GRDB.swift.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |s|
 	
 	s.ios.deployment_target = '8.0'
 	s.osx.deployment_target = '10.9'
+	s.tvos.deployment_target = '9.0'
```

Comment:
How did you pick that version?

Reply:
`9.0` is the first public version of the tvOS platform – https://en.wikipedia.org/wiki/tvOS#History